### ISSUE TITLE
Partial block comment in C++ not working

### ISSUE THREAD

**Issue by Anteru** [pygments/pygments-migration-test#821]
(Original issue 1114 created by renfeisong on 2015-05-28T05:23:45.906555+00:00)

When only `/*` appears, the code followed by `/*` will remain uncommented. For example,

```
#!cpp

#include <iostream>
using namespace std;
int main() { /*
    int a = 1;
    int b = 2;
    printf("Oh.");

    return -1;
}
```

I believe the conventionally way is to comment out all code after `/*`, *until* a `*/` is found.

Demo: [http://pygments.org/demo/1716577/](http://pygments.org/demo/1716577/)

**Comment by Anteru** [comment]
(Original issue 1114 last updated on 2016-06-01T04:46:57.939186+00:00)

(Issue automaticaly closed due to status in Bitbucket: resolved)

**Comment by Anteru** [comment]
(Original comment by tshatch on 2016-06-01T04:46:57.928766+00:00)

Allow open c-style comments.

Fixes #1114

→ <<cset 5a8a05059fc7>>
